src/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/main.scss" >
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar__month-length--28">
+      <div
+        class="calendar__cell calendar__cell--1 calendar__cell--start-day--wed"
+      ></div>
```

Comment:
```suggestion
      <div class="calendar calendar--start-day-mon calendar--month-length-31">
```

This should be main div which will align items based on the **calendar--start-day-mon** and **calendar--month-length-31**